tests/framework/e2e/etcd_process.go

```diff
@@ -329,6 +331,27 @@ func AssertProcessLogs(t *testing.T, ep EtcdProcess, expectLog string) {
 	}
 }
 
+func GetLeader(t *testing.T, epc *EtcdProcessCluster) EtcdProcess {
```

Comment:
I recall that we have an existing function with name like `WithLeader`?

Reply:
correct. I updated to use `epc.WaitLeader()`.